### PR TITLE
koji_import plugin will add userdata to brew build metadata

### PR DESCRIPTION
**Pull request by rcerven**
* CLOUDBLD-7986

Signed-off-by: Robert Cerven <rcerven@redhat.com>

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Changes to metadata also update the documentation for the metadata
- n/a Pull request has a link to an osbs-docs PR for user documentation updates
- [x] New feature can be disabled from a configuration file

### CODE REVIEW

**Review by mkosiarc — APPROVED** [review]

File: atomic_reactor/plugins/exit_koji_import.py [hunk 1]
```diff
@@ -99,8 +99,10 @@ class KojiImportBase(ExitPlugin):
     key = None
     is_allowed_to_fail = False
 
+    args_from_user_params = map_to_user_params("userdata")
+
     def __init__(self, workflow, blocksize=None,
-                 target=None, poll_interval=5):
+                 target=None, poll_interval=5, userdata=None):
```
Comment (chmeliik) [inline atomic_reactor/plugins/exit_koji_import.py:105]: Could the 'target' param be removed? I was confused about where it is coming from if not user params, but it turns out it is unused anyway

File: atomic_reactor/plugins/exit_koji_import.py [hunk 2]
```diff
@@ -727,9 +733,9 @@ class KojiImportSourceContainerPlugin(KojiImportBase):
     key = PLUGIN_KOJI_IMPORT_SOURCE_CONTAINER_PLUGIN_KEY  # type: ignore
 
     def __init__(self, workflow, blocksize=None,
-                 target=None, poll_interval=5):
+                 target=None, poll_interval=5, userdata=None):
         super(KojiImportSourceContainerPlugin, self).__init__(workflow, blocksize,
-                                                              target, poll_interval)
+                                                              target, poll_interval, userdata)
```
Comment (chmeliik) [inline atomic_reactor/plugins/exit_koji_import.py:738]: The inits in both subclasses could be removed

**Review by chmeliik — COMMENTED** [review]
LGTM, just saw an opportunity for a little bit of cleanup

File: atomic_reactor/plugins/exit_koji_import.py (hunk 1 above)
Comment (rcerven) [reply atomic_reactor/plugins/exit_koji_import.py:105]: sure will remove

File: atomic_reactor/plugins/exit_koji_import.py (hunk 2 above)
Comment (rcerven) [reply atomic_reactor/plugins/exit_koji_import.py]: done

**Review by chmeliik — APPROVED** [review]

**Pull request merged**